src/charts/bar.js

```diff
@@ -12,6 +12,7 @@ export default function Bar (_container) {
     },
     width: 800,
     height: 500,
+    chartId: null,
```

Comment:
We should continue trying to keep the config object free of internals that the user should not have to care about. That's one thing that makes mapd-charting so wrong. I think your idea of generating a UID is better. I used it multiple times. You could have a simple shared UID generator, say in common.js, that could initialize a random number or string once (for example in chart.js `if (!cache.svg)`) and would not have to pass it down as a config but the marks like bar and line could use it directly from common.js.